asab/zookeeper/container.py

```diff
@@ -92,7 +91,7 @@ def __init__(self, path, data):
 	async def _do_advertise(self, zoocontainer):
 		async with self.Lock:
 			if self.Node is not None and await zoocontainer.ZooKeeper.exists(self.Node):
-				await zoocontainer.ZooKeeper.set_data(self.Path, self.Data)
+				await zoocontainer.ZooKeeper.set_data(self.Node, self.Data)
```

Comment:
@ateska It was advertised to wrong node. 
I will test till evening to make sure I do not see the exception again.